posix/unix.c

```diff
@@ -393,16 +405,18 @@ int unix_accept4(unsigned socket, struct sockaddr *address, socklen_t *address_l
 			proc_threadWait(&s->queue, &s->spinlock, 0, &sc);
 
 		LIST_REMOVE(&s->connect, conn);
-		s->state &= ~US_ACCEPTING;
+		s->state &= ~(US_ACCEPTING | US_CAN_ACCEPT);
```

Comment:
If connect was done before accept, it would have set the US_CAN_ACCEPT flag on remote. I was only unsetting this flag here. But yes, now that I've corrected the handling to use `s->connect`, it is not needed now